javalib/src/main/scala/java/util/TimerTask.scala

```diff
@@ -0,0 +1,30 @@
+package java.util
+
+import scala.scalajs.js.timers
+
+abstract class TimerTask {
+  private[util] var cancelled: Boolean = false
+  private[util] var lastScheduled: Long = 0L
+
+  def run(): Unit
+
+  def cancel(): Unit = {
+    cancelled = true
+  }
```

Comment:
Since we have no weak reference mechanism in JavaScript to lazily clean up the `tasks` queue, `cancel()` should actually remove this task from the `Timer.tasks` queue eagerly.